test/service/test_response.py

```diff
@@ -2449,9 +2445,8 @@ def entity_types(self) -> List[str]:
     def test_empty_response(self):
         for entity_type in self.entity_types():
             with self.subTest(entity_type=entity_type):
-                url = furl(url=self.base_url,
-                           path=('index', entity_type),
-                           query_params={'order': 'asc'})
+                url = self.base_url.set(path=('index', entity_type),
+                                        query_params={'order': 'asc'})
```

Comment:
Using `query_params` here is fine, but make sure that either `args` or `query_params` (they are synonymous) are used consistently.